dfc/target.go

```diff
@@ -426,7 +426,7 @@ existslocally:
 	written, err := io.CopyBuffer(w, file, buf)
 	if err != nil {
 		errstr = fmt.Sprintf("Failed to send file %s, err: %v", fqn, err)
-		t.invalmsghdlr(w, r, errstr)
+		t.invalmsghdlr(w, r, errstr, false)
```

Comment:
Nice catch but it looks kind of hacky :) And if one wants skip writing to header and set error code it does not help. Maybe just replace this line with two:
```
glog.Errorln(errstr)
t.statsif.add("numerr", 1)
```
? In case of 'false' almost all of the invalmsghdlr does nothing

Reply:
1. Invalmsghdlr traces only caller. So, moving glog to here is the same as keep calling invalmsghdlr with false - in both cases glog ouputs target.go:429. As for URL - you can print it in-place :)
2. I do not think changing a function to fit only one corner-case call is better than replacing the call with 2 lines of code for that corner case